test/unit/api/user.spec.ts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-unused-expressions */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import 'chai/register-should';
+import jwt from 'jsonwebtoken';
+import app, { resetDatabase } from '../../../server-test';
+import { JWT_SECRET } from '../../../server/config';
+import * as modelUtils from '../../util/modelUtils';
+import { Errors as emailErrors } from '../../../server/routes/updateUserEmailInterval';
+import { Errors as updateEmailErrors } from '../../../server/routes/updateEmail';
+import { ConfigurationServicePlaceholders } from 'aws-sdk/lib/config_service_placeholders';
+
+const ethUtil = require('ethereumjs-util');
+chai.use(chaiHttp);
+const { expect } = chai;
+const markdownThread = require('../../util/fixtures/markdownThread');
+
+describe('User Model Routes', () => {
+  before('reset database', async () => {
```

Comment:
We don't need to add descriptions to these, I don't think they ever show up?

Reply:
right, I didn't know if the description is useful for code-auditor if/when something does go wrong with the tests.